src/main/kotlin/com/beust/kobalt/app/remote/RemoteDependencyData.kt

```diff
@@ -46,6 +47,8 @@ class RemoteDependencyData @Inject constructor(val executors: KobaltExecutors, v
         val buildFileCompiler = buildFileCompilerFactory.create(buildSources, pluginInfo)
         val projectResult = buildFileCompiler.compileBuildFiles(args)
 
+        val buildFileDependencies = Kobalt.buildFileClasspath.map {toDependencyData(it, "compile")}
```

Comment:
@cbeust The main problem here is that build file classpath can be changed: in case if we call the server to resolve dependencies for different `Build.kt` files. So we cannot store it in one static variable `Kobalt.buildFileClasspath`